src/content/admin/cleanup.mdx

```diff
@@ -20,15 +20,26 @@ Learn how to configure this in the [configuration steps](#configuration-steps-).
 
 Okteto considers an application or resources inactive if a user hasn't performed any of the following tasks during the `inactivity` period:
 
-- Upgrade or redeploy via the UI or the command line.
-- Launched a development container with `okteto up`.
-- Deploy the latest version of your code with `okteto deploy`.
+- Upgrade or redeploy via the UI or the command line
+- Launched a development container with `okteto up`
+- Deploy the latest version of your code with `okteto deploy`
 
 Service accounts, config maps, secrets, or volumes you create will be unaffected by this operation.
 
-## Wake sleeping resources
+### Events That Count as Activity to Reset the Sleep Counter
```

Comment:
This events, and the previous one below `Okteto considers an application or resources inactive if a user hasn't performed any of the following tasks during the `inactivity` period` are the same.
Each time there is any this events, we considered there is activity, and we reset the sleep counter.

Reply:
I called them out separately since the content was slightly different. I also feel like a user searching for one thing may not necessarily read the whole section. Happy to consolidate though if you feel like it's too much

Reply:
I think they are the same. We are explaining what keeps a namespace awaken. It's the same events.

Reply:
Ok, I shifted the content a bit to support this. Let me know what you think

Reply:
@pchico83 The latest changes should fix the duplication. Let me know if that looks alright to you